public/modules/custom/helfi_rekry_content/helfi_rekry_content.module

```diff
@@ -243,3 +244,44 @@ function _helfi_rekry_content_lookup_organization_tid($external_id) {
 
   return reset($ids);
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function helfi_rekry_content_preprocess_block(&$variables) {
+
+  if ($variables["plugin_id"] !== 'hero_block') {
+    return;
+  }
+
+  // Get current entity and entity version.
+  $entity_matcher = \Drupal::service('hdbt_content.entity_version_matcher')->getType();
+
+  /** @var \Drupal\Core\Entity\ContentEntityInterface $entity */
+  $entity = $entity_matcher['entity'];
+
+  // No need to continue if current entity doesn't have has_hero field.
+  if (
+    !$entity instanceof ContentEntityInterface ||
+    !$entity->hasField('field_has_hero')
+  ) {
+    return;
+  }
+
+  if ((bool) $entity->get('field_has_hero')->value) {
+    if (
+      in_array($entity->getType(), ['landing_page', 'page']) &&
+      isset($entity->get('field_content')->first()->entity)
+    ) {
+
+      $first_paragraph_gray = &$variables['content']['hero_block']['#first_paragraph_grey'];
+
+      // Check if the content field first paragraph is District and project
+      // search or District listing and add classes accordingly.
```

Comment:
Remove the District and project related comments :)

Reply:
Nice catch! Fixed :)